### PR TITLE
Adding Travis CI for building and testing the project.

### PR DESCRIPTION
**Pull request by jgebal**

### CODE REVIEW

**Comment by javornikolov** [comment]
Thank you @jgebal, it would be awesome to have CI hooked to our projects!

@rsim, seems some rights are needed to setup Travis-CI hook (me and Jaceck don't see the Settings tab). Could you please help!

File: .travis/setup_accounts.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+"$ORACLE_HOME/bin/sqlplus" -L -S / AS SYSDBA <<SQL
+alter user hr identified by hr account unlock;
+grant execute on dbms_lock to hr;
+exit
+SQL
+
+"$ORACLE_HOME/bin/sqlplus" -L -S / AS SYSDBA <<SQL
+create user arunit identified by arunit;
+grant dba to arunit;
```
Comment (javornikolov) [inline .travis/setup_accounts.sh]: Hmm. Do we really need dba role?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - ORACLE_FILE=oracle11g/xe/oracle-xe-11.2.0-1.0.x86_64.rpm.zip
+    - ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe
+    - NLS_LANG=AMERICAN_AMERICA.AL32UTF8
+    - ORACLE_BASE=/u01/app/oracle
+    - LD_LIBRARY_PATH=$ORACLE_HOME/lib
+    - PATH=$PATH:$ORACLE_HOME/jdbc/lib
+    - DATABASE_VERSION=11.2.0.2
+    - ORACLE_SID=XE
+    - DATABASE_NAME=XE
+    - TZ='Europe/London'
```
Comment (javornikolov) [inline .travis.yml]: What about `ORA_SDTZ`?

Comment (jgebal) [inline .travis.yml]: Will give it a try

File: .travis/setup_accounts.sh (hunk 1 above)
Comment (jgebal) [inline .travis/setup_accounts.sh]: fixed

File: .travis.yml (hunk 2 above)
Comment (jgebal) [inline .travis.yml]: After trying - we need TZ to run on db server.
The tests run fine in such env, we dont need ORA_SDTZ in this case, which is client-specific time-zone parameter

**Comment by javornikolov** [comment]
Looks good to me now :+1:

**Pull request merged**